src/types.ts

```diff
@@ -56,6 +56,16 @@ export type AutomoxDevice = {
     VENDOR: string;
     VERSION: string;
   };
+  status?: {
+    device_status?: string;
+    agent_status?: string;
+    policy_status?: string;
+    policy_statuses?: { id: number }[];
```

Comment:
What is the difference between these two fields? I'm surprised to see `policyStatus` as a string, and `policyStatuses` as numbers.

Reply:
I typed this from the actual response, this is how it looks like: 
![Screen Shot 2023-04-24 at 16 40 24](https://user-images.githubusercontent.com/13191932/234098911-e016b98e-f25f-47b6-9e75-7d87ca8a166d.png)

Reply:
Oh, I see. Should we type like this then?
```suggestion
    policy_status?: string;
    policy_statuses?: { id: number, compliant: boolean }[];
```

Reply:
sure!